x-pack/plugins/security_solution/public/entity_analytics/api/api.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RISK_SCORE_PREVIEW_URL } from '../../../common/constants';
+
+import { KibanaServices } from '../../common/lib/kibana';
+import type { GetScoresResponse } from '../../../server/lib/risk_engine/types';
+import type { RiskScorePreviewRequestSchema } from '../../../common/risk_engine/risk_score_preview/request_schema';
+
+/**
+ * Fetches preview risks scores
+ */
+export const fetchPreviewRiskScore = async ({
```

Comment:
Nit: I've been referring to this as a _risk score preview_ as opposed to a _preview risk score_. The distinction is probably entirely subjective, but probably best to be consistent.